src/translations/ach_UG.po

```diff
@@ -2612,7 +2612,7 @@ msgstr "crwdns38045:0crwdne38045:0"
 msgid "Client withdrawals prior to this year amounted to almost USD 500 million. More than USD 140 million worth of client withdrawals is expected this year."
 msgstr "crwdns43616:0crwdne43616:0"
 
-msgid "Client's demo accounts will be deleted after 365 days of inactivity."
+msgid "Client's demo accounts will be deleted after 30 days of inactivity."
```

Comment:
@salar-fs please don't update the autogenerated translations files directly. they will be auto updated once we run the update translations script after merging your PR. @reza-fs @mahdiaryayi-fs please assist if there's any confusion